programs/ssl/ssl_client2.c

```diff
@@ -131,6 +131,7 @@ int main( void )
 #define DFL_ETM                 -1
 #define DFL_CA_CALLBACK         0
 #define DFL_EAP_TLS             0
+#define DFL_REPRODUCIBLE        0
```

Comment:
Please add `reproducible` to `USAGE`. Applies to the server as well.

Reply:
ok, did I place it correctly Amon the other ones ?